ddtrace/contrib/_trace_utils_llm.py

```diff
@@ -208,14 +204,23 @@ def trunc(self, text):
             text = text[: self._config.span_char_limit] + "..."
         return text
 
-    def llm_record(self, span, attrs):
-        # type: (Span, Dict[str, Any]) -> None
+    @classmethod
+    @abc.abstractmethod
+    def _llmobs_tags(cls, span: Span) -> List[str]:
+        """Generate a list of llmobs tags from a given span."""
+        return []
+
+    def llm_record(self, span: Span, attrs: Dict[str, Any], tags: Optional[List[str]] = None) -> None:
         """Create a LLM record to send to the LLM Obs intake."""
         if not self.llmobs_enabled:
             return
-        llm_record = {}
-        if span is not None:
-            llm_record["dd.trace_id"] = str(span.trace_id)
-            llm_record["dd.span_id"] = str(span.span_id)
-        llm_record.update(attrs)
-        self._llmobs_writer.enqueue(llm_record)
+        llmobs_tags = self._llmobs_tags(span)
+        if span is not None and span.sampled:
+            # FIXME: this is a temporary workaround until we figure out why 128 bit trace IDs are stored as decimals.
+            llmobs_tags.insert(0, "dd.trace_id:%s" % hex(span.trace_id)[2:])
```

Comment:
We can use format to generate hex. This is cleaner than the built in hex() function.

```suggestion
            llmobs_tags.insert(0, "dd.trace_id:%s" %  '{:x}'.format(span.trace_id))
```